os/arch/arm/Kconfig

```diff
@@ -263,6 +263,13 @@ config ARCH_ROMPGTABLE
 	---help---
 		Support a fixed memory mapping use a (read-only) page table in ROM/FLASH.
 
+config DEBUG_SYSTEM_FAULTS
```

Comment:
By this config, only one debug line will be enabled or be disabled. It does not affect performance and does not print much debug message. And it's necessary message to debug a crash.
I don't know this is necessary. I mean we can use lldbg instead of fltlldbg without this config.

Reply:
agree to this, may be it can be removed, as in crash case this minimal logs are mandatory

Reply:
Done. Using lldbg instead of fltlldbg. Removed the newly added config.